extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfigPersistenceUnit.java

```diff
@@ -310,13 +310,52 @@ interface HibernateOrmConfigPersistenceUnitDialect {
          *
          * E.g. `MyISAM` or `InnoDB` for MySQL.
          *
+         * @deprecated Use {@code mysql.}{@linkplain MySQLDialectConfig#storageEngine storage-engine} instead
+         *
          * @asciidoclet
```

Comment:
I am not sure using Javadoc tags such as `@code` and `@linkplain` will work with `@asciidoclet`... Though it probably doesn't matter in this specific case, as deprecated configuration properties are not included in documentation.